reconciler/webutils.py

```diff
@@ -121,7 +121,7 @@ def parse_raw_results(input_keys, response):
                 current_df.drop(["features"], axis=1, inplace=True)
                 current_df["type_id"] = [item[0]["id"] for item in current_df["type"]]
                 current_df["type"] = [item[0]["name"] for item in current_df["type"]]
-            except IndexError:
+            except KeyError:
```

Comment:
As discussed on the issue, this would be better:

```suggestion
            except (IndexError, KeyError):
```

Reply:
Thanks.  I've made this change.  There seems to be a white space change that sneaked in there.  If you'd like I can try to remove it, but it's in the doc.